src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
```

Comment:
```suggestion
        return Colors.values()[index].name();
```
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/81781479/203402420-16301454-5ee1-4925-bb66-c1cdef39f935.png">